seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobExecutorServiceImpl.java

```diff
@@ -63,6 +65,14 @@
 @Slf4j
 @Service
 public class JobExecutorServiceImpl implements IJobExecutorService {
+    public static final String DIR_PROFILE = "profile";
+    public static final String USER_DIR = getUserDir();
+    public static final String DOT_CONF = ".conf";
+
+    private static String getUserDir() {
```

Comment:
why you extracted getUserDir method. what is the purpose? can you make it constant?